app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.age = age
+        self.name = name
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    pupil_array = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = \
```

Comment:
Better not to use ` \` as a linebreaker, create temporary variable instead or use parentheses
While both `\` and parentheses can be used for line continuation, parentheses are generally recommended for their clarity, readability, and predictability